pom.xml

```diff
@@ -53,42 +49,46 @@
 		</pluginRepository>
 	</pluginRepositories>
 	<dependencies>
-		<dependency>
-			<groupId>xml-apis</groupId>
-			<artifactId>xml-apis</artifactId>
-			<version>1.4.01</version>
-		</dependency>
-  		<dependency>
-    		<groupId>org.eclipse.jetty</groupId>
-    		<artifactId>jetty-server</artifactId>
-    		<version>9.4.25.v20191220</version>
-    		<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.jetty</groupId>
-			<artifactId>jetty-util</artifactId>
-			<version>9.4.25.v20191220</version>
-		</dependency>
-		<dependency>
-			<groupId>javax.servlet</groupId>
-			<artifactId>javax.servlet-api</artifactId>
-			<version>4.0.1</version>
-		</dependency>	
 		<dependency>
 			<groupId>org.ow2.clif</groupId>
 			<artifactId>clif-api</artifactId>
 			<version>3.0.1</version>
+		 <exclusions>
```

Comment:
Generally means that runtime behavior is hard to predict. You can try

* manual sanity checks
* `RealJenkinsRule`
* `pluginFirstClassLoading` (also dangerous)
* using a different artifact that has fewer deps
* forking a tool rather than trying to load it inside the controller’s JVM